lua/wire/stools/light.lua

```diff
@@ -32,8 +31,7 @@ if SERVER then
 			self:GetClientNumber("r"),
 			self:GetClientNumber("g"),
 			self:GetClientNumber("b"),
-			self:GetClientNumber("spritesize"),
-			self:GetClientNumber("on") ~= 0
```

Comment:
Keep these because it lets the user choose whether it starts on or off.

Reply:
Should this persist between duplications then? ~~Thus nullifying my effort in removing this~~

Reply:
Yes, the light's on state should persist between duplications.